src/activities/number_click/Number_click.qml

```diff
@@ -0,0 +1,604 @@
+                    onExited: { parent.color = "#00A31B" }
+                    onPressed: {
+                        buttonTick.height = parent.height * 0.7
+                        activity.audioEffects.play("qrc:/gcompris/src/core/resource/sounds/bleep.wav")
+                        Activity.checkAnswer()
+                    }
+                    onReleased: { buttonTick.height = parent.height * 0.6 }
+                }
+            }
+        }
+
+        /* Set the error panel */
+        Rectangle {
+            id: errorpanel
+
+            visible: false
+
+            height: parent.height * 0.6
+            width : parent.width * 0.6
+
+            color: "white"
+            border.width: 4
+            border.color: "black"
+            radius: 30
+
+            anchors.horizontalCenter: parent.horizontalCenter
+            anchors.verticalCenter: parent.verticalCenter
+
+            MouseArea {
+                id: errorpanelArea
+
+                anchors.fill: parent
+                hoverEnabled: true
+
+                onEntered: {  }
+                onExited: {  }
+                onPressed: {
+                    activity.audioEffects.play("qrc:/gcompris/src/core/resource/sounds/bleep.wav")
+                    errorpanel.visible = false
+                }
+                onReleased: {  }
+            }
+
+            Rectangle {
+                id: correctAnswer
+
+                height: tenthbar.height * 1.05
+                width: firstbar.width * 1.25
+
+                anchors.verticalCenter: parent.verticalCenter
+                //anchors.horizontalCenter: firstbar.horizontalCenter
+
+                border.color: "green"
+                border.width: 7
+                radius: 10
+            }
+
+            Rectangle {
+                id: wrongAnswer
+
+                height: tenthbar.height * 1.05
+                width: firstbar.width * 1.25
+
+                anchors.verticalCenter: parent.verticalCenter
+                //anchors.horizontalCenter: tenthbar.horizontalCenter
+
+                border.color: "red"
+                border.width: 7
+                radius: 10
+            }
+
+            Rectangle {
+                id: firstbar
+
+                height: parent.height * 0.09 //0.09 * 1
+                width: parent.width * 0.07
+
+                color: "grey"
+                border.width: 2
+                border.color: "black"
+                radius : 10
+
+                anchors.left: parent.left
+                anchors.leftMargin: parent.width * 0.05
+                anchors.bottom: parent.bottom
+                anchors.bottomMargin: parent.height * 0.05
+
+                Text {
+                    id: firstbar_text
+
+                    text: "1"
+                    color: "white"
+                    style: Text.Outline
+                    styleColor: "black"
+
+                    font.pointSize: 24 * background.height / 1080
```

Comment:
I'm not sure what the 1080 means, it should probably be a variable or use some defined values in https://invent.kde.org/education/gcompris/-/blob/master/src/core/GCText.qml#L38